packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-dropdown/gux-form-field-dropdown.tsx

```diff
@@ -108,25 +109,29 @@ export class GuxFormFieldDropdown {
   render(): JSX.Element {
     return (
       <GuxFormFieldFieldsetContainer labelPosition={this.computedLabelPosition}>
-        <GuxFormFieldLegendLabel
-          position={this.computedLabelPosition}
-          required={this.required}
-          labelText={this.label?.textContent}
+        <GuxFormFieldLegendLabelContainer
+          labelPosition={this.computedLabelPosition}
         >
-          <slot name="label" onSlotchange={() => this.setLabel()} />
-          {this.renderScreenReaderText(
-            this.getI18nValue('required'),
-            this.required
-          )}
-          {this.renderScreenReaderText(
-            getSlotTextContent(this.root, 'error'),
-            this.hasError
-          )}
-          {this.renderScreenReaderText(
-            getSlotTextContent(this.root, 'help'),
-            this.hasHelp
-          )}
-        </GuxFormFieldLegendLabel>
+          <GuxFormFieldLegendLabel
+            required={this.required}
+            labelText={this.label?.textContent}
+          >
+            <slot name="label" onSlotchange={() => this.setLabel()} />
```

Comment:
Can we do ` onSlotchange={this.setLabel}`

Reply:
Is there a benefit here other than brevity?

Reply:
Yeah, just brevity. No worries if you want to keep it. Just wanted to see what you thought.